### DISCUSSION TITLE
Error: "Unable to find fields.json for sdk version [eforms-sdk-0.4] on s3 bucket."

### DISCUSSION THREAD

**Discussion by vlad-alien-dev** [OP-TED/eForms-SDK#282] [Q&A] [answered]
https://viewer.ted.europa.eu/swagger-ui/index.html#/notice-render-controller/renderNotice
When sending a request for rendering to HTML/PDF, an error is returned:
{
  "errorCode": 400,
  "message": "Unable to find fields.json for sdk version [eforms-sdk-0.4] on s3 bucket.",
  "timestamp": "2022-12-21T08:35:08Z"
}
Seems to be a bug.

**Comment by pcolast** [comment, marked as answer]
Hello,

What was the value of CustomizationID of the notice you requested the rendering ?
Please note that only SKD version >= 1.0.0 are supported.

If this didn't solve the issue, could you please upload a file with the full request body that you tried and was answered with the error 400 ? 
Thank you.

Best regards,
Stefan
